include/dyno/vtable.hpp

```diff
@@ -78,8 +78,10 @@ struct local_vtable<boost::hana::pair<Name, boost::hana::basic_type<Signature>>.
   constexpr explicit local_vtable(ConceptMap map)
     : vtbl_{boost::hana::make_map(
       boost::hana::make_pair(Name{}, detail::erase_function<Signature>(map[Name{}]))...
-    )}
-  { }
+    )} {
+    // suppress "unused" warnings for empty parameter packs
```

Comment:
Oh, so this warning is caused by _empty_ parameter packs. I was wondering. That's with GCC, right? Seems like a QOI issue (although it's okay for us to work around it).